go.mod

```diff
@@ -175,7 +173,8 @@ require (
 )
 
 replace (
-	github.com/argoproj-labs/argocd-image-updater/registry-scanner => ../registry-scanner
+	// Uncomment for local testing
+	// github.com/argoproj-labs/argocd-image-updater/registry-scanner => ./registry-scanner/
```

Comment:
Added a comment for stating that we need to uncomment this line for local testing if we have changes in registry-scanner module.